src/app/index.js

```diff
@@ -1 +1,6 @@
 import "./index.css";
+
+const github = document.getElementById("github");
+if (github.getAttribute("href") === "") {
+  github.remove();
+}
```

Comment:
I think this could be done more elegantly with CSS.

```css
.sub-link[href=""] {
  display: none;
}
````

Reply:
way better, thanks!